# The problem contains start and goal 
# 
geometry_msgs/Pose2D start
geometry_msgs/Pose2D goal
